spring-cloud-kubernetes-core-k8s-client/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>spring-cloud-kubernetes</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>2.0.0-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-kubernetes-core-k8s-client</artifactId>
```

Comment:
artifact and package name seem odd as well. Maybe just `client` since its the official kubernetes client. The other being fabric8?

Reply:
That's what I suggested here https://github.com/spring-cloud/spring-cloud-kubernetes/pull/661#issuecomment-707900166. Maybe `*-kubernetes-client-autoconfig` and `*-kubernetes-fabric8-autoconfig`? Not sure. Repeating k8s seems odd. The official artifact is `java-client` which isn't helpful either.